addons/point_of_sale/static/src/app/models/related_models.js

```diff
@@ -683,6 +692,10 @@ export function createRelatedModels(modelDefs, modelClasses = {}, indexes = {})
     function loadData(rawData, load = []) {
         const results = {};
         const missingRecords = {};
+        const callbacks = mapObj(processedModelDefs, () => ({
+            updated: new Map(),
+            created: new Map(),
+        }));
```

Comment:
There is already an array of "callbacks" per model in this file. Now, here, you are introducing a new local variable the masks the original. Why don't you use the old one?

Reply:
I'm referring to this: https://github.com/odoo/odoo/blob/bb92c588ccd06e4f80daf4878cdfa0799cb1ab35/addons/point_of_sale/static/src/app/models/related_models.js#L182

Reply:
Hello that's normal, isn't the same use :)